package.json

```diff
@@ -40,7 +50,11 @@
     "chai": "4.0.0",
     "mocha": "7.2.0",
     "mocha-standard": "1.0.0",
+    "rollup": "^4.24.0",
     "semistandard": "11.0.0",
     "snazzy": "7.0.0"
+  },
+  "dependencies": {
+    "inherits-ex": "^2.1.0"
```

Comment:
We fixe the versions because not everyone is using semver the same way (unfortunately)